api/model.go

```diff
@@ -396,6 +401,10 @@ type UpdateSnapshotMaxSize struct {
 	SnapshotMaxSize string `json:"snapshotMaxSize"`
 }
 
+type UpdateReplicaRebuildingBandwidthLimit struct {
```

Comment:
Why do we have two structures that have the same content?
`UpdateReplicaRebuildingBandwidthLimitInput`

Reply:
`UpdateReplicaRebuildingBandwidthLimitInput` is for the schema  in `func NewSchema()client. Schemasas`